### DISCUSSION TITLE
[Solved] Do Not Install Chromebrew for Any Reason!!!

### DISCUSSION THREAD

**Discussion by CroutonIsFun** [dnschneid/crouton#5065] [General]
EDIT: Chromebrew is working fine. I'm not sure what the problem was earlier. Strange...

@dnschneid @DennisLfromGA @drinkcat 

Until the recent ChromeOS update, I had been using Chromebrew to install crew_sudo, which provides sudo in the crosh shell.

It has now become impossible to install Chromebrew correctly. After entering the install command, the routine crashes after only a few lines, leaving the command line in an unusable state: every command now gives a segmentation fault.

Note that the only fix is a powerwash, followed by setting up your Chromebook from scratch.

Some very clever people may know a way to fix this, but for now, Chromebrew has become another victim of the relentless ChromeOS development.

For reference, my current version is 123.0.6312.112.

Fortunately for crouton fans, crouton still works quite well as long as you know a few tricks, such as setting up sshd inside the chroot, then ssh into your chroot from the crosh shell. This is a tribute to the work of David Schneider, who so thoroughly understands ChromeOS, that even to this day, crouton is usable!

**Comment by DennisLfromGA** [comment]
@CroutonIsFun,
 I've never been a big fan of ChromeBrew, being a big brunch advocate I always install the [brunch-toolchain](https://github.com/sebanc/brunch-toolchain) which gives me more than I need. You can install it in chromeOS even without brunch and it works great.

-**DennisLfromGA**

**Reply by CroutonIsFun** [reply]
Thanks, I'll have a look.

**Comment by CroutonIsFun** [comment]
Will reopen if appropriate...
